app/views/accounts/login.html.erb

```diff
@@ -0,0 +1,14 @@
+<h1>Please sign in below</h1>
+<% if @errors %>
+<% @errors.each do |message| %>
+  <%= message %>
+<%end%>
+<%end%>
```

Comment:
Nice work applying this P2 pattern. You can now do `flash.each`... if you use the Flash hash